cmd/geth/main.go

```diff
@@ -336,9 +336,6 @@ func prepare(ctx *cli.Context) {
   5. Networking is disabled; there is no listen-address, the maximum number of peers is set
      to 0, and discovery is disabled.
 `)
-
-	case !ctx.IsSet(utils.NetworkIdFlag.Name):
-		log.Info("Starting Geth on BSC mainnet...")
```

Comment:
why remove this log?

Reply:
> why remove this log?

otherwise, when begin syncing agaist chapel, will output
`Starting Geth on BSC mainnet...`

Reply:
> why remove this log?

otherwise, when begin syncing agaist chapel, will output
`Starting Geth on BSC mainnet...`